src/js-tracker.js

```diff
@@ -15,17 +15,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
-var request = require('request');
+var axios = require('axios');
 var moment = require('moment');
+var uuidv4 = require('uuid').v4;
+var ScalableBloomFilter = require('bloom-filters').ScalableBloomFilter;
+
+// by default it creates an ideally scalable bloom filter for 8 elements with an error rate of 0.01 and a load factor of 0.5
+var filter = new ScalableBloomFilter();
```

Comment:
The bloomfilter it only required in the backend server. It is better not have this dependency in a potential js client.

Reply:
fixed